compose/cli/docker_client.py

```diff
@@ -9,27 +8,9 @@ def docker_client():
     Returns a docker-py client configured using environment variables
     according to the same logic as the official Docker client.
     """
-    cert_path = os.environ.get('DOCKER_CERT_PATH', '')
-    if cert_path == '':
-        cert_path = os.path.join(os.environ.get('HOME', ''), '.docker')
```

Comment:
From what I can see `kwargs_from_env()` doesn't include this default for a cert path, is that correct?  Is this path no longer used?